### ISSUE TITLE
Feature request: Multiple Choice Questions for Forms 

### ISSUE THREAD

**Issue by itkorai** [nextcloud/forms#227]
**Which Forms version are you running?** (see apps page)
1.1.1
**Nextcloud or ownCloud?:**
Nextcloud 
**Nextcloud/ownCloud version:** (see Nextcloud admin page)
17
### Steps to reproduce
1. When creating the Form, there are some answers we can put like options
2. While Putting the answers make 1 field as Best Answer , if someone choose Option A, B , C ,D
it shows if A is correct and person choose B option it should show Incorrect and shows that the Option A is correct and show some explanation content  that we can choose upon adding the question
3. at the end shows how many question a user has answered as correct and shows explanation on each question and answer!

**Comment by skjnldsv** [comment]
Hi!
I really don't understand this issue, could you try differently?
You would like conditional question? Like if you answered yes to question A, you go to question B otherwise question C ?

**Comment by szaimen** [comment]
> Hi!
> I really don't understand this issue, could you try differently?
> You would like conditional question? Like if you answered yes to question A, you go to question B otherwise question C ?

I think he would like to use forms to create a quiz for other users.

But what you proposed sound also like a really good idea :)

**Comment by skjnldsv** [comment]
xD, I think it's another level we're not ready to reach

**Comment by skjnldsv** [comment]
@jancborchardt , we want this?

**Comment by jancborchardt** [comment]
Has come up when interviewing other people about Forms before, so yes eventually it would be nice to have. No priority at all though, setting it to "Maybe someday".

**Comment by 3tyh86qvt** [comment]
that's a  pity, though. I'm looking throughout the Internet to find a substitute for Google docks and nothing seems to offer the  option to make quizzes :/
I wish I could help with programming

**Comment by jancborchardt** [comment]
@3tyh86qvt yep, understandable and sorry that this is not a priority yet. It’s simply because we are still early in development of the app and need to get the basics rock solid first.
And anyone who would like to look into it can dive in to developing this. :)

@3tyh86qvt a big help would actually be if you can post screenshots of how e.g. Google Forms does this.

**Comment by ei8fdb** [comment]
> Hi!
> I really don't understand this issue, could you try differently?
> You would like conditional question? Like if you answered yes to question A, you go to question B otherwise question C ?

This conditional logic would be an important feature for creating user research questionnaires/surveys. I know that's not part of this GH issue tho'. :)

**Comment by jancborchardt** [comment]
Let’s be clear: This issue is about the quizzing aspect, not about the conditional logic. The issue for conditional logic is #358

Microsoft Forms has a quiz functionality which might be good to look at for this.

**Comment by AlessandroDM** [comment]
Hello Everyone,
In case this is still being considered, 
+1 on this request / Ability to create Quizzes in Nextcloud using Forms.
I was looking at replacing https://www.flexiquiz.com/ with Nextcloud forms but unfortunately there aren't yet enough functionalities at the moment of writing this.

In addition of being able to define correct answers, it would be nice to have the ability to end up the Quiz with a score rating
(in percentage or "Pass / Fail" or number of correct answers / wrong answers).

The www.flexiquiz.com web site allows you to see (for free without even an account needed) how it works and what options are available (it also has a neat "Fill the blanks" system).

This can be useful for Training, Legal, Health and Safety and so many more use-cases.

Thank you.

**Comment by hyoretsu** [comment]
This would be great for people moving from Google Drive to Nextcloud.

**Comment by n3snah** [comment]
+1 to this. I would also like to have some kind of quiz functionality. Forms seems like the best app for this too rather than creating a new app which will be almost identical with some extra logic.
